package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
```

Comment:
Should we increase the version? @tonygcs @DionisioRD

Reply:
We didn't do it before so we can keep it. I don't have a hard preference anyway.

Reply:
we could change it to 1.0.0 or 2.0.0 to align it with SICAP but it is not very important as Antonio said, it is just a demo

Reply:
I know, anyway, we could do it to have them synchronized.